### DISCUSSION TITLE
Bad link to nvim-cmp in 'Features' page

### DISCUSSION THREAD

**Discussion by beatmax** [NvChad/NvChad#2417] [General]
In the 'features' page:

[nvim-cmp](A completion plugin for Neovim coded in Lua.) is a completion plugin for Neovim coded in Lua.

**Comment by siduck** [comment]
thx for reporting. fixed https://github.com/NvChad/nvchad.github.io/commit/1aed2116c22be8b7dc0cc9fbe6db71b0a935f8a9
